### ISSUE TITLE
ListView in Xamarin Forms 4 With Shell in Android not Scrolling

### ISSUE THREAD

**Issue by YovanyTabares** [xamarin/Xamarin.Forms#6381]
### Description
ListView in Xamarin Forms 4 With Shell in Android not Scrolling

### Steps to Reproduce

1. Fill a List View With Data
2. Try Scrolling 

### Expected Behavior
ListView Scrolling

### Basic Information

- Version with issue: Xamarin Forms 4.0

- Platform Target Frameworks: 
  - Android: API 19, 22

**Comment by samhouts** [comment]
Same root cause as #5306 and #5205

**Comment by King-Xero** [comment]
Just ran into this issue, seems to be somewhat of a result of the following code being added to `ListViewRenderer.cs` in #4522:
```
if (Forms.IsLollipopOrNewer)
    nativeListView.NestedScrollingEnabled = true;
```

For now i'm using this solution as a workaround. If you create a custom renderer for your `ListView` and add the following code as an override:
```
protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
{
        base.OnElementChanged(e);
        if (Control == null) return;
        Control.NestedScrollingEnabled = false;
}
```

Hope this helps someone.

**Comment by samhouts** [comment]
#6310

**Comment by moustafashaban** [comment]
IsPullToRefresh=false, solved the issue, but I hope the team can solve that soon

**Comment by PureWeen** [comment]
@adrianknight89 @moustafashaban @YovanyTabares 

I wasn't able to recreate this issue. That being said
https://github.com/xamarin/Xamarin.Forms/pull/7032

Should fix whatever ListView woes you are having. If you have a reproduction of your issue with ListView not scrolling that would be helpful to ensure that I did indeed address your scenario.

I tested creating a basic ListView and I tested by setting IsPullToRefresh to true and I never encountered any scrolling issues
